react/src/components/SourceCodeViewer.tsx

```diff
@@ -0,0 +1,87 @@
+import Flex from './Flex';
+import { CheckOutlined, CopyOutlined } from '@ant-design/icons';
+import { Button } from 'antd';
+import { Highlight, themes } from 'prism-react-renderer';
+import { useRef, useState } from 'react';
+// @ts-ignore
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+
+interface SourceCodeViewerProps {
+  // In case of using this component in a web component, the props should be parsed from the web component.
+  codeBlock: string;
+  language: string;
+  fontSize?: string;
+  minHeight?: string;
+}
+
+const SourceCodeViewer: React.FC<SourceCodeViewerProps> = ({
+  codeBlock,
+  language,
+  fontSize,
+  minHeight,
```

Comment:
I removed that props.
I thought it to provide the control over the UI of the code viewer, but it would be more good to have the code viewer use the height and width of the parent and control the UI in the component that uses the code viewer.